kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -61,8 +49,6 @@
   public static final String ID_DEFAULT = "";
 
   public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
-  private static final String MAX_POLL_RECORDS_DOC =
-          "The maximum number of records returned in a single call to poll().";
```

Comment:
why remove this doc string?

Reply:
while we're in the rest proxy config, do you want to depreciate `PRODUCER_THREADS_CONFIG` given its not used? and maybe add to the doc string saying that it's not? (maybe a separate PR)

Reply:
1. max.poll.records is not configurable in REST Proxy. This value is used as is.
2. Done.